packages/gizmo/src/Rotate.ts

```diff
@@ -381,36 +367,37 @@ export class RotateControl extends GizmoComponent {
   }
 
   private _updateAxisTransform(): void {
-    this._cameraPos.copyFrom(this._camera.entity.transform.position);
-    this._tempMatrix.getRotation(this._tempQuat);
-
-    const { _tempQuat: tempQuat } = this;
-
-    Quaternion.invert(tempQuat, tempQuat);
-    Vector3.transformByQuat(this._cameraPos, tempQuat, this._cameraPos);
-
-    Quaternion.rotationX(-Math.atan2(this._cameraPos.y, this._cameraPos.z), tempQuat);
-    this._axisX.transform.rotationQuaternion = tempQuat;
-    this._axisXHelper.transform.rotationQuaternion = tempQuat;
-
-    Quaternion.rotationY(Math.atan2(this._cameraPos.x, this._cameraPos.z), tempQuat);
-    this._axisY.transform.rotationQuaternion = tempQuat;
-    this._axisYHelper.transform.rotationQuaternion = tempQuat;
-
-    Quaternion.rotationZ(Math.atan2(this._cameraPos.y, this._cameraPos.x), tempQuat);
-    this._axisZ.transform.rotationQuaternion = tempQuat;
-    this._axisZHelper.transform.rotationQuaternion = tempQuat;
-
-    this._localLookAt(this._axisXYZ.transform, this._cameraPos);
-    this._localLookAt(this._axisXYZHelper.transform, this._cameraPos);
+    const { _tempMat: _tempMat, _tempVec, _tempVec2, _cameraPos } = this;
+    // 相机位置
+    _cameraPos.copyFrom(this._camera.entity.transform.worldPosition);
+    const gizmoTrans = this.gizmoEntity.transform;
+    // 获取参照向量(Gizmo 中点 -> 相机)（世界坐标）
+    Vector3.subtract(_cameraPos, gizmoTrans.worldPosition, _tempVec);
+    // 逆矩阵（世界 -> Gizmo）
+    Matrix.invert(gizmoTrans.worldMatrix, _tempMat);
+    // 将向量(Gizmo 中点 -> 相机)映射到 Gizmo 坐标系中，此处 XYZ 局部向量是相同的
+    Vector3.transformNormal(_tempVec, _tempMat, _tempVec2);
+    const factor = MathUtil.radToDegreeFactor;
+    const { x, y, z } = _tempVec2;
+    // 用 yoz 投影计算 X 轴的局部旋转
+    this._axisX.transform.rotation.x = -Math.atan2(y, z) * factor;
+    this._axisXHelper.transform.rotation.x = -Math.atan2(y, z) * factor;
+    // 用 xoz 投影计算 Y 轴的局部旋转
+    this._axisY.transform.rotation.y = Math.atan2(x, z) * factor;
+    this._axisYHelper.transform.rotation.y = Math.atan2(x, z) * factor;
```

Comment:
this._axisY.transform.rotation.y = this._axisYHelper.transform.rotation.y = Math.atan2(x, z) * factor;

Reply:
done